src/test/regress/expected/subselect_gp.out

```diff
@@ -3238,30 +3240,32 @@ with cte as
 (update t1 set i = 1
  returning *)
 select * from t2
-where t2.i in (select i from cte where t2.i = cte.i);
-                                      QUERY PLAN                                      
---------------------------------------------------------------------------------------
+where t2.i in (select i from cte where t2.i = cte.i and cte.i > 0);
+                                         QUERY PLAN                                         
+--------------------------------------------------------------------------------------------
  Gather Motion 3:1  (slice3; segments: 3)
    ->  Seq Scan on t2
          Filter: (SubPlan 1)
          SubPlan 1  (slice3; segments: 3)
-           ->  Subquery Scan on cte
+           ->  Result
                  Filter: (t2.i = cte.i)
                  ->  Materialize
                        ->  Broadcast Motion 3:3  (slice2; segments: 3)
-                             ->  Update on t1
```

Comment:
Do we really need a full set (Insert, Delete, Update) of DML operations to test? It seems, that the patch modify all of them in the same approach.

Reply:
They are redundant, right.